code/modules/vending/engivend.dm

```diff
@@ -15,7 +15,12 @@
 					/obj/item/electronics/apc = 10,
 					/obj/item/electronics/airalarm = 10,
 					/obj/item/electronics/firealarm = 10,
-					/obj/item/electronics/firelock = 10)
```

Comment:
This should be the last item on the list, dump the new stuff into the second-last slot.